### PR TITLE
Replace internal ELB URLs with their public counterparts.

### PR DESCRIPTION
**Pull request by ndouglas**
## Description
In several places, the codebase uses some long, tall, and _ugly_ URLs for CMS environments, like `internal-dsva-vagov-prod-cms-2000800896.us-gov-west-1.elb.amazonaws.com`, tbh a hostname only a mother could love.

Why the codebase does this to us is not known; perhaps it dates from the time of GitHub-hosted runners; perhaps it goes back further, to the time of Jenkins.  Perhaps the answers are lost in the mists of antiquity.

Anyway, I'm opening this PR to see if I can do away with this nonsense, because a security audit wants us to use HSTS on everything and I don't think AMZN is gonna let us generate TLS certs with their hostnames.

There's a decent amount of work left to do, not least of which is me poring over the code and trying to understand what is happening and why it's happening to me, but I wanted to open this draft PR early and get some comments in case this is dum and doomed to failure.

See https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9217

**TL;DR**: `http://internal-dsva-vagov-prod-cms-2000800896.us-gov-west-1.elb.amazonaws.com/ -> https://prod.cms.va.gov/`

## Testing done
none lol, lemme get on that.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by rmessina1010 — APPROVED** [review]
LOKTM.

**Review by ryguyk — APPROVED** [review]
This appears to do what you say it's doing. I'm not sure if that is going to work or not, but I think I understand that you are trying to answer that very question.

**Review by CBonade — APPROVED** [review]
All checks pass, LGTM!
